src/number_theory/legendre_symbol/auxiliary.lean

```diff
@@ -162,6 +324,30 @@ begin
   exact (h₁ b hb.symm),
 end
 
+/-- The trace map from a finite field of characteristic `p` to `zmod p` -/
+-- These two lemmas depend on `ring_theory.trace` and finite fields.
+-- SUGGESTION: move them to the end of `ring_theory.trace` (this would likely
```

Comment:
I checked that neither of the two files imports the other.
An alternative could be to add a file `field_theory/finite/trace.lean` containing the definition and lemma. (I assume they could be useful; otherwise I can also move them to the file on additive characters which uses them).
Preferences?

Reply:
See [this Zulip topic](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Add.20import.20to.20existing.20mathlib.20file.20or.20create.20a.20new.20one.3F/near/285575518).
A new file seems to be the way to go.